tests/test_runtime.py

```diff
@@ -325,7 +324,7 @@ def test_ansatz_qiskit(self, token, tol, shots):
             backend="ibmq_qasm_simulator",
             hamiltonian=hamiltonian,
             ansatz="EfficientSU2",
-            x0=[3.97507603, 3.00854038],
+            x0=[0.0, 0.0, 0.0, 0.0],
```

Comment:
How come this was changed?

Reply:
We are using EfficientSu2 from the Qiskit library and it has 4 parameters when there are 2 qubits.

Reply:
We are using EfficientSu2 from the Qiskit library and it has 4 parameters when there are 2 qubits.